### PR TITLE
implemented app uniformity in title bars and layout

### PR DESCRIPTION
**Pull request by mahikaw**
The title bar in all activities of the app have been removed and now the app has a uniform layout all throughout. Addresses #220 .This is ready to be merged. @chhavip please review and merge. Thanks :smile:

### CODE REVIEW

File: PowerUp/app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -4,13 +4,16 @@
         Base application theme, dependent on API level. This theme is replaced
         by AppBaseTheme from res/values-vXX/styles.xml on newer devices.
     -->
-    <style name="AppBaseTheme" parent="Theme.AppCompat.Light">
+        <style name="MyNoActionBar" parent="Theme.AppCompat.Light.NoActionBar">
```
Comment (chhavip) [inline PowerUp/app/src/main/res/values/styles.xml]: Give this a better name than MyNoActionBar. Be more generic.

File: PowerUp/build.gradle [hunk 2]
```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.2.3'
+        classpath 'com.android.tools.build:gradle:2.1.3'
```
Comment (chhavip) [inline PowerUp/build.gradle]: Do not change gradle version in this PR. If it is to be done, it will be done in a separate PR.

File: PowerUp/gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -1,6 +1,6 @@
-#Thu Sep 24 10:14:11 CEST 2015
+#Tue Dec 20 11:28:28 IST 2016
 distributionBase=GRADLE_USER_HOME
```
Comment (chhavip) [inline PowerUp/gradle/wrapper/gradle-wrapper.properties]: Remove this file.

File: PowerUp/.idea/gradle.xml [hunk 4]
```diff
@@ -12,6 +12,12 @@
             <option value="$PROJECT_DIR$/app" />
           </set>
```
Comment (chhavip) [inline PowerUp/.idea/gradle.xml]: Do not commit your .idea or .iml files. Please remove all of these.

**Comment by chhavip** [comment]
@mahikaw , reviewed and changes are required.

File: PowerUp/app/src/main/AndroidManifest.xml [hunk 5]
```diff
@@ -18,35 +19,44 @@
...
             android:name=".AvatarActivity"
-            android:label="@string/app_name" />
+            android:label="@string/app_name"
+            android:theme="@style/MyNoActionBar"/>
         <activity
             android:name=".MapActivity"
             android:label="@string/app_name"
-            android:launchMode="singleTop" />
+            android:launchMode="singleTop"
+            android:theme="@style/MyNoActionBar"/>
```
Comment (chhavip) [inline PowerUp/app/src/main/AndroidManifest.xml]: Is this required in all activities? Change theme of only those as indicated in the issue.

Comment (mahikaw) [inline PowerUp/app/src/main/AndroidManifest.xml]: @chhavip I created an abstract theme for all the activities so that it is easier to incorporate any changes in theme design in future.

Comment (mahikaw) [inline PowerUp/app/src/main/AndroidManifest.xml]: Should I remove it nonetheless?

**Comment by mahikaw** [comment]
Hi @chhavip, I've added the changes you asked for but I request you to merge the gitignore PR  #213 as everyone has to remove the build changes in the .idea and .iml files manually everytime. thanks :smile:

File: PowerUp/.idea/vcs.xml [hunk 6]
```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (chhavip) [inline PowerUp/.idea/vcs.xml]: This is still not required.

File: PowerUp/app/src/main/AndroidManifest.xml [hunk 7]
```diff
@@ -18,35 +19,44 @@
...
         <activity
             android:name=".SelectFeaturesActivity"
-            android:label="@string/app_name"/>
+            android:label="@string/app_name"
+            android:theme="@style/AppBaseTheme"/>
         <activity
             android:name=".ScenarioOverActivity"
-            android:label="@string/app_name"/>
+            android:label="@string/app_name"
+            android:theme="@style/AppBaseTheme"/>
```
Comment (chhavip) [inline PowerUp/app/src/main/AndroidManifest.xml]: There is no need to change the theme of each activity and can be done by changing the base theme of the application in the manifest. As the current AppTheme inherits from the AppBaseTheme where you have specified the changes, I do not think specification is required in each activity declaration.

Comment (mahikaw) [inline PowerUp/app/src/main/AndroidManifest.xml]: yes, sure. I will work on it right away.

**Comment by chhavip** [comment]
What are the problems you are facing as you have not fixed the PR as per the required comments?

**Comment by mahikaw** [comment]
So this file vcs.xml  is not in my repo, as it has been apparently deleted. So to fix this I tried copying the exact same vcs.xml in my repo from the main branch and committing it, however it is not tracking my changes for this particular file. Please help me solve this. Thanks :smile:

**Comment by chhavip** [comment]
Rebase to the latest version of the project and see if the problem persists. If it still does, I request you to create a fresh branch with the changes and submit again

**Comment by mahikaw** [comment]
I have created a PR from a new branch as the vcs.xml wasn't getting updated. 
closing this one
